### PR TITLE
Add the NodeVertical Job from workloads repo

### PR DESCRIPTION
**Pull request by akrzos**
@chaitanyaenr PTAL and advise.

### CODE REVIEW

**Comment by akrzos** [comment]
@chaitanyaenr PTAL

**Comment by chaitanyaenr** [comment]
@akrzos created a PR for adding baseline workload to the pipeline, we can use this as an example.

**Comment by akrzos** [comment]
@chaitanyaenr Can you see if the source control is properly added with this job, also job status is now checked at the end and should provide pass/fail.

File: jjb/dynamic/workloads_nodevertical.yml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    builders:
+    - shell: |+
+        set -o pipefail
+        set -eux
+
+        git clone https://${SSHKEY_TOKEN}@github.com/redhat-performance/perf-dept.git
+        export PUBLIC_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
+        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
+        chmod 600 ${PRIVATE_KEY}
+
```
Comment (chaitanyaenr) [inline jjb/dynamic/workloads_nodevertical.yml]: cd ${WORKSPACE}/workloads?

File: pipeline-scripts/nodevertical.groovy [hunk 2]
```diff
@@ -21,73 +21,71 @@ stage ('nodevertical_scale_test') {
...
-						[$class: 'StringParameterValue', name: 'TOOLING_INVENTORY', value: tooling_inventory_path ],
-						[$class: 'BooleanParameterValue', name: 'CLEAR_RESULTS', value: Boolean.valueOf(clear_results) ],
-						[$class: 'BooleanParameterValue', name: 'MOVE_RESULTS', value: Boolean.valueOf(move_results) ],	
-						[$class: 'BooleanParameterValue', name: 'USE_PROXY', value: Boolean.valueOf(use_proxy) ],
-						[$class: 'StringParameterValue', name: 'PROXY_USER', value: proxy_user ],
-						[$class: 'StringParameterValue', name: 'PROXY_HOST', value: proxy_host ],
-						[$class: 'StringParameterValue', name: 'ENVIRONMENT', value: env ],
-						[$class: 'StringParameterValue', name: 'GITHUB_TOKEN', value: token ],
-						[$class: 'BooleanParameterValue', name: 'CONTAINERIZED', value: Boolean.valueOf(containerized) ]]
+                        [$class: 'StringParameterValue', name: 'CLUSTER_USER', value: cluster_user ],
```
Comment (chaitanyaenr) [inline pipeline-scripts/nodevertical.groovy]: spacing issue.

**Pull request merged**